### PR TITLE
doc: fixing typos

### PR DESCRIPTION
**Pull request by deining**
This PR fixes typos in two files (`README.md`, `dagre.js`.
It also converts some links from `http` to `https`.

### CODE REVIEW

File: d2layouts/d2dagrelayout/dagre.js [hunk 1]
```diff
@@ -232,11 +232,11 @@ var cc=0;_.forEach(southEntries.forEach(function(entry){var index=entry.pos+firs
  * constraint graph this function will resolve any conflicts between the
  * constraint graph and the barycenters for the entries. If the barycenters for
  * an entry would violate a constraint in the constraint graph then we coalesce
- * the nodes in the conflict into a new node that respects the contraint and
+ * the nodes in the conflict into a new node that respects the constraint and
  * aggregates barycenter and weight information.
  *
  * This implementation is based on the description in Forster, "A Fast and
- * Simple Hueristic for Constrained Two-Level Crossing Reduction," thought it
+ * Simple Heuristic for Constrained Two-Level Crossing Reduction," thought it
```
Comment (alixander) [inline d2layouts/d2dagrelayout/dagre.js:239]: oops looks like one more, "though it"

**Review by alixander — APPROVED** [review]
thank you!

File: d2layouts/d2dagrelayout/dagre.js (hunk 1 above)
Comment (deining) [reply d2layouts/d2dagrelayout/dagre.js]: > oops looks like one more, "though it"

Fixed now.

**Comment by alixander** [comment]
@deining we require commits to be signed, could you please get that set up on your account?

**Comment by deining** [comment]
> @deining we require commits to be signed, could you please get that set up on your account?

The commit is signed off now. I hope this is what you wanted.

**Comment by alixander** [comment]
Screen Shot 2022-12-27 at 12 04 38 PM

i'm still seeing this. i don't think it worked

when commits are signed you see this verified badge

Screen Shot 2022-12-27 at 12 05 55 PM

**Comment by deining** [comment]
> i'm still seeing this. i don't think it worked

You are right, it didn't work with my previous commit.
 
> when commits are signed you see this verified badge

My latest commit shows the verified bag now and the warning is gone :smile:. Looking forward to see this PR merged!

**Comment by alixander** [comment]
thx for bearing with that @deining , it's a one-time thing, so if you decide to contribute more, this will be a nonissue =)

**Pull request merged**